packages/@lwc/integration-karma/test/component/LightningElement.attachInternals/api/index.spec.js

```diff
@@ -13,74 +13,91 @@ const testConnectedCallbackError = (elm, msg) => {
     expect(error.message).toBe(msg);
 };
 
-const createTestElement = (name, def) => {
-    const elm = createElement(name, { is: def });
-    document.body.appendChild(elm);
-    return elm;
+const createElementsThroughLwcAndCustomElementConstructor = (tagName, ctor) => [
+    createElement(`ai-${tagName}`, { is: ctor }),
```

Comment:
What is `ai-`?

Reply:
I was going for an abbreviation of attach internals 😆 , might be a bit confusing though.